frontend/src/app/modules/queue-dashboard/components/location-info-list/location-info-list.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, Input} from '@angular/core';
+import {LocationInfoDTO} from "../../models/LocationInfoDTO";
+import {ServerInfo} from "../../services/queue-dashboard.service";
+import {OsmLangService} from "../../../../services/osm-lang.service";
+import {DatePipe} from "@angular/common";
+import {local} from "d3-selection";
+
+@Component({
+  selector: 'osm-queue-list',
+  templateUrl: './location-info-list.component.html',
+  styleUrls: ['./location-info-list.component.scss']
+})
+export class LocationInfoListComponent {
+
+  @Input() wptServerID;
+
+  private langservice: OsmLangService;
+  locationInfo : LocationInfoDTO[] = [];
+
+  constructor(langservice: OsmLangService) {
```

Comment:
If you use 
```
  constructor(private langservice: OsmLangService) {
```
you can skip the declaration of the member variable above and the assignment below.